### PR TITLE
Update Japanese Language JSON.

### PR DESCRIPTION
**Pull request by reindex-ot**
but app's UI has not been tested.

### CODE REVIEW

File: mobile/src/modules/i18n/translations/ja.json [hunk 1]
```diff
@@ -10,98 +10,98 @@
...
-      "translate": "Help translate the app in other languages.",
-      "code": "Report issues, contribute, or request new features.",
-      "thirdParty": "What makes this app possible."
+      "appUpdate": "最新のバージョンが公開されています！",
+      "appearance": "アプリの外観をカスタマイズします。",
+      "backup": "デバイス間でデータを移動します。",
+      "insights": "アプリの使用状況と統計。",
+      "saveErrors": "一部のトラックが保存されなかった理由を確認します。",
+      "interactions": "Nothing でサポートされている機能",
+      "scanning": "音楽のディレクトリのオプション",
```
Comment (cyanChill) [inline mobile/src/modules/i18n/translations/ja.json:84]: Is `音楽のディレクトリのオプション` suppose to translate to `Music directory options` (original was `Music discovery options.`)?

Also we're missing the `。` at the end for this (`scanning`) and in the `interactions` above.

File: mobile/src/modules/i18n/translations/ja.json [hunk 2]
```diff
@@ -10,98 +10,98 @@
...
 
     "description": {
-      "backup": "Transfer data from this app across devices through a `music_backup.json` file.",
-      "listAllow": "Defaults to the following if empty:",
-      "listBlock": "Has higher priority over the entries in the Allow List.",
-      "ignoreDuration": "Duration in seconds of audio we don't want to save."
+      "backup": "｢music_backup.json｣ファイルを通じて、このアプリからデバイス間でデータを転送します。",
+      "listAllow": "空の場合はデフォルトで次のようになります:",
+      "listBlock": "許可リスト内のエントリよりも優先されます。",
+      "ignoreDuration": "保存をしない音楽の再生時間 (秒数)"
```
Comment (cyanChill) [inline mobile/src/modules/i18n/translations/ja.json:95]: Missing `。` at the end of `ignoreDuration` for consistency.

File: mobile/src/modules/i18n/translations/ja.json [hunk 3]
```diff
@@ -113,8 +113,8 @@
       "total": "合計",
       "totalDuration": "合計時間",
 
-      "pathAdd": "Add directory.",
-      "pathSelect": "Select directory."
+      "pathAdd": "ディレクトリを追加",
```
Comment (cyanChill) [inline mobile/src/modules/i18n/translations/ja.json:116]: Missing `。` at the end of `pathAdd` for consistency.

File: mobile/src/modules/i18n/translations/ja.json [hunk 4]
```diff
@@ -124,102 +124,102 @@
     "missing": "なし",
     "from": "From",
```
Comment (cyanChill) [inline mobile/src/modules/i18n/translations/ja.json:125]: This is untranslated; unsure if there's no good translation for `From` in Japanese?

**Review by cyanChill — CHANGES_REQUESTED** [review]
Besides a question for the translation for the `settings.brief.scanning` key, missing translation for the `errorScreen.from` key, and some missing periods, everything looks good!

File: mobile/src/modules/i18n/translations/ja.json (hunk 4 above)
Comment (reindex-ot) [reply mobile/src/modules/i18n/translations/ja.json:125]: I don't know where `From` is used.

Comment (cyanChill) [reply mobile/src/modules/i18n/translations/ja.json:125]: It's used on the "Unmatched Route" screen; we display the route we couldn't find and show the route we were previously on in the "From".

**Comment by reindex-ot** [comment]
Fixed.

**Review by cyanChill — APPROVED** [review]

**Pull request merged**